DuckDuckGo/Subscription/Subscription.xcassets/Exclamation-Color-16.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Exclamation-Color-16.svg",
```

Comment:
Not sure if it causes a problem but we use PDF assets everywhere.

Reply:
I'll fix this in the next PR